perception/map_based_prediction/include/map_based_prediction_node.hpp

```diff
@@ -0,0 +1,200 @@
+// Copyright 2021 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef MAP_BASED_PREDICTION_NODE_HPP_
```

Comment:
MAP_BASED_PREDICTION__MAP_BASED_PREDICTION_NODE_HPP_

Reply:
done

Reply:
Same reason as above

Reply:
not resolved

Reply:
This one is also cannot changed because of the pre-commiter

Reply:
done